apis/compute/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -280,6 +280,16 @@ func (in *GKEClusterParameters) DeepCopyInto(out *GKEClusterParameters) {
 		*out = make([]string, len(*in))
 		copy(*out, *in)
 	}
+	if in.NetworkRef != nil {
```

Comment:
Hmm I may be missing something but it was surprising to me that values in `apis/compute` generated files would change here?

Reply:
See the commit history - I also did not expect this so I left it in a separate commit. I just now confirmed that if I run `make reviewable` on stack-gcp master I get the content of this commit.

Reply:
Gotcha, thanks!